caasp-multi-master-environment.yaml

```diff
@@ -0,0 +1,19 @@
+---
+parameters:
+  root_password: linux
+  admin_flavor: m1.large
+  master_flavor: m1.large
+  worker_flavor: m1.large
+  worker_count: 1
+  master_count: 1
+  external_net: ext-net
+  # Use this lbaas provider for master api:
+  # octavia (default) OR haproxy
+  master_lb_provider: octavia
```

Comment:
I think better is go for haproxy as default, since it is more common

Reply:
Agreed. Fixed